src/coffee/stockimport.coffee

```diff
@@ -371,8 +371,18 @@ class StockImport
             .fetch()
             .then (result) =>
               @_updateInventory(entry, result.body, tryCount + 1)
+            .catch (err) =>
+              if (err.statusCode == 404)
+                debug "Stock not existing anymore (probably was deleted), create a new stock"
+                @client.inventoryEntries.create(entry)
+              else
+                debug "Error on refetching stock on 409 error. Details: #{JSON.stringify(err)}"
```

Comment:
`Error on handling 409 stock update error`. You do not know if it was fetching or updating so it might be anything like 500 or 400

Reply:
https://github.com/sphereio/sphere-stock-import/pull/69/commits/f0227ce01505b32c5565cc6d784389281b01533d